project-base/storefront/components/Layout/Header/MenuIconic/MenuMyAccountList.tsx

```diff
@@ -0,0 +1,84 @@
+import { MenuIconicSubItemLink } from './MenuIconicElements';
+import { SalesRepresentative } from './SalesRepresentative';
+import { EditIcon } from 'components/Basic/Icon/EditIcon';
+import { ExitIcon } from 'components/Basic/Icon/ExitIcon';
+import { SearchListIcon } from 'components/Basic/Icon/SearchListIcon';
+import { UserIcon } from 'components/Basic/Icon/UserIcon';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import { TIDs } from 'cypress/tids';
+import useTranslation from 'next-translate/useTranslation';
+import { useCurrentCustomerUserPermissions } from 'utils/auth/useCurrentCustomerUserPermissions';
+import { useLogout } from 'utils/auth/useLogout';
+import { getInternationalizedStaticUrls } from 'utils/staticUrls/getInternationalizedStaticUrls';
+
+export const MenuMyAccountList: FC = () => {
+    const { t } = useTranslation();
+    const logout = useLogout();
+    const { canManageUsers } = useCurrentCustomerUserPermissions();
+    const { url } = useDomainConfig();
+    const [customerOrdersUrl, customerComplaintsUrl, customerEditProfileUrl, customerUsersUrl] =
+        getInternationalizedStaticUrls(
+            ['/customer/orders', '/customer/complaints', '/customer/edit-profile', '/customer/users'],
+            url,
+        );
+
+    const userMenuItemTwClass =
+        'h-14 rounded-xl bg-backgroundAccentLess border border-background hover:bg-background hover:border-borderAccentLess';
+    const userMenuItemIconTwClass = 'flex flex-row w-[44px] justify-start';
+
+    return (
+        <ul className="flex flex-col max-h-[87dvh] overflow-auto gap-2 p-1">
+            <li className={userMenuItemTwClass}>
+                <MenuIconicSubItemLink href={customerOrdersUrl} tid={TIDs.header_my_orders_link} type="orderList">
+                    <div className={userMenuItemIconTwClass}>
+                        <SearchListIcon className="w-6 h-6" />
```

Comment:
When the width and height are the same we can use `size-6`, you can use it for all icons in this list.

Reply:
Thanks, fixes.